rllib/connectors/tests/test_connector.py

```diff
@@ -61,6 +70,28 @@ def test_sanity_check(self):
         self.assertEqual(m[0], [m.connectors[0]])
         self.assertEqual(m[m.connectors[1].__class__], [m.connectors[1]])
 
+    def test_pipeline_indexing(self):
+        """Tests if ConnectorPipeline.__getitem__() works as intended."""
+        context = ConnectorContext({}, observation_space=gym.spaces.Box(-1, 1, (1,)))
+        some_connector = MeanStdObservationFilterAgentConnector(context)
+        some_other_connector = ObsPreprocessorConnector(context)
+        # Create a dummy pipeline just for indexing purposes
+        pipeline = ConnectorPipeline(context, [some_connector, some_other_connector])
+
+        for key, expected_value in [
+            (MeanStdObservationFilterAgentConnector, [some_connector]),
+            ("MeanStdObservationFilterAgentConnector", [some_connector]),
+            (SyncedFilterAgentConnector, [some_connector]),
+            ("SyncedFilterAgentConnector", []),
+            (ClipRewardAgentConnector, []),
+            ("can i get something?", []),
+            (0, [some_connector]),
+            (1, [some_other_connector]),
+        ]:
+            print(key)
+            print(expected_value)
```

Comment:
cool. should we get rid of the prints?
if assertEqual fails, it will print the actual and expected values.

Reply:
Changed! Thanks